src/css/styles.scss

```diff
@@ -220,16 +220,18 @@ button:focus {
   outline:0;
 }
 
-.all-cards {
+.all-cards, .all {
```

Comment:
I noticed that the styling could have been shared with `.all-cards` and remove unnecessary lines of code